File: Palindrom/Palindrom.java
package palindrom;

import java.util.Scanner;

public class Palindrom {

    public static void main(String[] args) {
        Scanner reader = new Scanner(System.in);
        System.out.print("Palindrome olup olmadığını kontrol etmek istediğiniz kelimeyi girin: ");
        String kelime = reader.nextLine();
        boolean kontrol = false;
        int uzunluk = kelime.length();
        for (int i = 0; i < uzunluk; i++) {
            if (kelime.charAt(i) != kelime.charAt(uzunluk - 1 - i)) {
                kontrol = true;
            }
        }
        if (kontrol == false) {
            System.out.println("Bu kelime palindromedur.");
        } else {
            System.out.println("Bu kelime palindrome değildir.");
        }
    }
}

File: Palindrom/Palindrom.py
def palindrom_mu(kelime):

    palindrom = True

    for i in range(int((len(kelime)/2))):
        if kelime[i] != kelime[len(kelime)-1-i]:
            palindrom = False

    if palindrom == True:
        print("{} palindromdur.".format(kelime))
    else:
        print("{} palindrom değildir.".format(kelime))

palindrom_mu(input("Bir kelime girin : "))